README.md

```diff
@@ -12,33 +12,42 @@ Editor (used in inspirehep.net) for JSON documents with bibliographic metadata
 
 ## Tech
 
-* Angular 2
+* Angular
 * Webpack
 * Sass
 * Typescript
 * Bootstrap 3
 * [ng2-json-editor](https://github.com/inveniosoftware-contrib/ng2-json-editor)
 
-## Quick start
+## Development with [inspire-next](https://github.com/inspirehep/inspire-next)
 
-```bash
-# clone our repo or alternatively your fork
-git clone https://github.com/inspirehep/record-editor.git
 
-# change directory to our repo
-cd record-editor
+### On `record-editor`
 
+```bash
 # install the repo with npm
 npm install
 
-# start the server
+# start build with watcher
 npm start
+
+# (on another shell) link the module
+npm link
```

Comment:
This needs to happen in this order? Or the `link` can be done first and then `npm start` ? To avoid having to use another shell

Reply:
yes this should work too, this is a left-over order for me from ng2-json-editor. :)